appengine/storage/appengine_config.py

```diff
@@ -0,0 +1,4 @@
+from google.appengine.ext import vendor
+
+# Add any libraries installed in the "lib" folder.
+vendor.add('lib')
```

Comment:
Hm... I thought I added this comment, but it doesn't seem to be here..

Can you update the README with installation notes about installing requirements.txt into lib?